jupyterhub/templates/proxy/autohttps/configmap.yaml

```diff
@@ -0,0 +1,106 @@
+{{- $HTTPS := (and .Values.proxy.https.hosts .Values.proxy.https.enabled) }}
+{{- $autoHTTPS := (and $HTTPS (eq .Values.proxy.https.type "letsencrypt")) }}
+{{- if $autoHTTPS -}}
```

Comment:
Hmmm, should we only have this configmap if we want autohttps as compared to https with a manually provided secret?

Reply:
Right now manually provided secrets are handled at CHP, so we only need this with autoHTTPS

Reply:
Wait what, is CHP doing TLS termination if the secret is manually provided?

Reply:
Yep, look at https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/jupyterhub/templates/proxy/deployment.yaml#L61. I think we should move it to traefik in a different PR

Reply:
Oh. This sums up my experience of the https matters in z2jh, a failure to get a good overview! I'll try catch up on this as quickly i can

Reply:
I created #1545 to represent this discussion.

Reply:
I created #1545 to represent this discussion.